nixpkgs_review/report.py

```diff
@@ -328,7 +328,7 @@ def print_console(self, pr: int | None) -> None:
         if pr is not None:
             pr_url = f"https://github.com/NixOS/nixpkgs/pull/{pr}"
             info("\nLink to currently reviewing PR:")
-            link(f"\u001b]8;;{pr_url}\u001b\\{pr_url}\u001b]8;;\u001b\\\n")
+            link(pr_url)
```

Comment:
```suggestion
            link(f"https://github.com/NixOS/nixpkgs/pull/{pr}")
```

Reply:
Rationale for this suggestion being?.. Either way seems fine, so unless there's a strong reason, I'd stick to status-quo.

Reply:
To the python virtual stack machine these two are nearly identical, but the former preserves `git blame` and may in the future see re-use